doc/lua_api.txt

```diff
@@ -2469,8 +2469,9 @@ and `minetest.auth_reload` call the authetification handler.
 * `minetest.add_item(pos, item)`: Spawn item
     * Returns `ObjectRef`, or `nil` if failed
 * `minetest.get_player_by_name(name)`: Get an `ObjectRef` to a player
-* `minetest.get_objects_inside_radius(pos, radius)`
+* `minetest.get_objects_inside_radius(pos, radius)`: returns a list of object references
     * `radius`: using an euclidean metric
+    * This function skips removed objects.
```

Comment:
Redundant IMO. We know the background story and thus, why this should be documented. But will assume this behaviour anyway, as they expect items to be removed instantly.

Reply:
See obj:remove(), it says that items are removed when returning from lua. So when modders use this function, call objects' remove and then call this function again with the same parameters, they know that the remove()d objects are not returned.